2200032/소득 불균형.py

```diff
@@ -0,0 +1,10 @@
+T = int(input())
+for test_case in range(1, T+1):
+    N = int(input())
+    lst = [int(x) for x in input().split()]
+    avg = sum(lst) // len(lst)
```

Comment:
좋은 풀이네요! 👍 
```suggestion
    avg = sum(lst) / len(lst)
```
정확한 평균 값을 구하려면 `//`가 아닌 `/` 연산을 활용해주시는 것이 맞습니다 😄